tests/openassetio-python/CMakeLists.txt

```diff
@@ -13,25 +20,12 @@ install(
     DESTINATION ${install_subdir}
 )
 
-# Set Python extension module name, and remove any ("lib") prefix.
-set_target_properties(openassetio-python-test
-    PROPERTIES
-    OUTPUT_NAME _openassetio_test
-    PREFIX ""
-    # Python modules on macOS must be .so rather than .dylib. But that
-    # means clang will complain:
-    #   invalid argument '-compatibility_version 1.0.0' only allowed
-    #   with '-dynamiclib'
-    # Similarly for '-current_version'.So unset the version flags. See
-    # also https://cmake.org/cmake/help/latest/prop_tgt/VERSION.html#mach-o-versions)
-    SOVERSION ""
-    VERSION "")
-
 if (WIN32)
-    set_target_properties(openassetio-python-test PROPERTIES SUFFIX ".pyd")
-    install(FILES $<TARGET_PDB_FILE:openassetio-python-test>
+    install(
```

Comment:
Not specifically about this, but since the changes in the tests almost mirror those in the OpenAssetIO Python module, but there's no PDB install for the latter that I can spot. Intentional?

Reply:
Windows and PDBs and install locations are still a bit of a mess.  We have an issue to look into this some more. Taking a snippet of the commit message:

> For Windows builds we currently install nearly all files twice, once under the bin/lib directories, and again under the Python `site-packages` directory. This was a hacky workaround to make Windows Python builds work (due to no RPATH).
> ...
> Tidying up the Windows build to avoid the duplication is the subject of future work, see https://github.com/OpenAssetIO/OpenAssetIO/issues/538.

So no not really intentional, just not properly considered.